Library/Tests/ViewModels/RewardPledgeViewModelTests.swift

```diff
@@ -585,27 +585,27 @@ internal final class RewardPledgeViewModelTests: TestCase {
         let paymentRequest: NSDictionary =  [
           "countryCode": project.country.countryCode,
           "currencyCode": project.country.currencyCode,
-          "merchantCapabilities": [PKMerchantCapability.Capability3DS.rawValue],
+          "merchantCapabilities": [PKMerchantCapability.capability3DS.rawValue],
           "merchantIdentifier": PKPaymentAuthorizationViewController.merchantIdentifier,
           "supportedNetworks": PKPaymentAuthorizationViewController.supportedNetworks,
-          "shippingType": PKShippingType.Shipping.rawValue,
-          "paymentSummaryItems": [
+          "shippingType": PKShippingType.shipping.rawValue,
+          "paymentSummaryItems": [[String: Any]].init(arrayLiteral:
```

Comment:
for some reason i couldn't use a regular array literal for this, i had to specify this initializer...